app/models/payment/go_cardless/transaction.rb

```diff
@@ -95,8 +96,8 @@ def publish_failed_subscription_charge
 
   def increment_funding_counter
     FundingCounter.update(page: page, currency: currency, amount: (begin
-                                                                     amount * 100
-                                                                   rescue
+                                                                     amount
+                                                                   rescue StandardError
                                                                      nil
```

Comment:
We don't need this check anymore if we're not multiplying (I added it initially because with GoCardless, sometimes amount is nil, and the multiplication would throw a NoMethodError exception).